context.go

```diff
@@ -164,10 +164,10 @@ func (c *Context) Load(key string) (interface{}, error) {
 // This function should only be used if the Load and Save functions are not sufficient.
 // Additionally, the user should not create buckets directly on the DB but always
 // call this function to create new buckets to avoid bucket name conflicts.
-func (c *Context) GetAdditionalBucket(name string) (*bolt.DB, string) {
-	fullName := c.bucketName + "_" + name
+func (c *Context) GetAdditionalBucket(name []byte) (*bolt.DB, []byte) {
```

Comment:
I'd be OK with this staying string, if you thought it makes the API nicer.

Reply:
Since bucketName is already []byte, I thought it'll be consistent for this one to be []byte too.

Reply:
OK, merging.